src/StreamJsonRpc/RpcMarshalableAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit: all the other attribute classes are declared in files under the `Reflection` folder. But the namespace you have is correct.